src/chatdlg.cpp

```diff
@@ -92,9 +92,12 @@ void CChatDlg::OnLocalInputTextTextChanged ( const QString& strNewText )
 
 void CChatDlg::OnSendText()
 {
-    // send new text and clear line afterwards
-    emit NewLocalInputText ( edtLocalInputText->text() );
-    edtLocalInputText->clear();
+    // send new text and clear line afterwards, do not send an empty message
+    if ( edtLocalInputText->text() != "" )
```

Comment:
I think there is an .empty() function for QString. I would use that instead.

Reply:
Thanks for pointing that out. I changed the code. I hope that is how you intended it